prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/etc/rest.custom_volume.yaml

```diff
@@ -0,0 +1,109 @@
+# merge:
+# counters:
+#   - ^comment => comment
+#   - hidden_fields:
+#     - comment
```

Comment:
What is this section? Maybe some kind of `TODO`?

It matches some seemingly inconsistent config below: `comment` is in the counters as well as in hidden_fields, is that okay?

Reply:
For ZAPI configuration, extending is easier. One could just add the changed fileds in a custom file, and harvest will merge them. However, for REST, the entire file needs to be replaced. I requested a feature [here.](https://github.com/NetApp/harvest/issues/2607). So the section is about what would be needed when the merging feature is available for REST. But good catch, the section only applies to ZAPI conf.

Reply:
the hidden_fields is for requesting extra fields that is not returned otherwise.
https://github.com/NetApp/harvest/blob/26e5f2700f928cee206206d1049cbbaa4b5a72ad/docs/configure-rest.md?plain=1#L164